cmd/verify/permissions/cmd.go

```diff
@@ -63,6 +64,10 @@ func run(cmd *cobra.Command, argv []string) {
 	ok, err := client.ValidateSCP()
 	if err != nil {
 		reporter.Errorf("Unable to validate SCP policies")
+		if strings.Contains(err.Error(), "Throttling: Rate exceeded") {
+			reporter.Errorf("%v. Please wait 3-5 minutes before retrying.", err)
```

Comment:
I would add a newline character between the `%v` and the message to wait:
`"%v\nPlease wait 3-5 min...."`

Reply:
Actually, scratch that - since we know that the issue was throttling, we don't need to output the `err` at all, so something like this should work:
```
if string.Contains... {
  reporter.Errorf("Throttling: Rate exceeded. Please wait 3-5 minutes before retrying.")
}
```

Reply:
@vkareh Please check now